lib/base-compiler.js

```diff
@@ -36,12 +36,16 @@ var child_process = require('child_process'),
     exec = require('./exec'),
     logger = require('./logger').logger,
     compilerOptInfo = require("compiler-opt-info"),
-    argumentParsers = require("./compilers/argument-parsers");
+    argumentParsers = require("./compilers/argument-parsers"),
+    cfg = require('./cfg');
 
 function Compile(compiler, env) {
     this.compiler = compiler;
     this.env = env;
     this.asm = new asm.AsmParser(env.compilerProps);
+    
+    this.cfg = new cfg.ControlFlowGraph({});
+    logger.log(cfg);
```

Comment:
Can you make this a `logger.debug` please? That way we get it if we run with `--debug` but I don't get log files filled with CFGs on the CE servers.

Reply:
done